SystemInfo/SystemInfoCollectorRegistry.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * File containing the SystemInfoCollectorRegistry class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo;
+
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector;
+
+/**
+ * A registry of SystemInfoCollectors that uses the SystemInfoCollector's service tag alias to identify the collector.
+ */
+class SystemInfoCollectorRegistry
+{
+    /** @var \EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector[] */
+    private $registry = [];
+
+    /**
+     * Adds a SystemInfoCollector with its service tag alias to the registry.
+     *
+     * @param \EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector $infoCollector
+     * @param string $alias Service tag alias
+     */
+    public function addToRegistry(SystemInfoCollector $infoCollector, $alias)
+    {
+        $this->registry[$alias] = $infoCollector;
+    }
+
+    /**
+     * Returns the SystemInfoCollector matching the argument.
+     *
+     * @param string $serviceTagAlias A service tag alias to match against. Example: composer
+     *
+     * @return \EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector|null The SystemInfoCollector, or null if there is none.
+     */
+    public function getFromRegistry($serviceTagAlias)
```

Comment:
Good, with that perspective it is generic enough that I could make an interface in front and have a specific "ServiceTag" implementation of it, solving the extensibility concern. (Well, apart from the fact that not all implementations may need an "identifier" parameter as it may be included in the object. But they could just pass false then.)

As for naming I'm being consistent with the viewbuilder registry :) But just ´getFrom´ sounds bad. How about `getItem`, or `get`?